packages/generator/src/lib/Generator/processPrismaField.ts

```diff
@@ -471,6 +471,12 @@ const processPrismaField = (
     options ??= {};
     options.template = template;
   }
+  const selectEnabled = directives.find(`enableSelect`);
+  if (selectEnabled !== undefined) {
+    options ??= {};
+    options.enableSelect = selectEnabled.tArgs[0];
+  }
```

Comment:
it seems that in most cases we've chosen to keep the snapshot term 'as is'.
`enableSelect` rather than `selectEnabled`

```suggestion
  const enableSelect = directives.find(`enableSelect`);
  if (enableSelect !== undefined) {
    options ??= {};
    options.enableSelect = enableSelect.tArgs[0];
  }
```
it seems easier if someone is looking for documentation regarding the term.